packages/aws-cdk/lib/init-templates/app/dotnet/src/HelloCdk/HelloConstruct.cs

```diff
@@ -0,0 +1,29 @@
+using Amazon.CDK;
+using Amazon.CDK.AWS.IAM;
+using Amazon.CDK.AWS.S3;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace HelloCdk
+{
+    public class HelloConstruct : Construct
+    {
+        readonly IList<Bucket> _buckets = new List<Bucket>();
+
+        public HelloConstruct(Construct parent, string id, HelloConstructProps props) : base(parent, id)
+        {
+            foreach (int i in Enumerable.Range(0, props.BucketCount))
+            {
+                _buckets.Add(new Bucket(this, $"Bucket{i}", new BucketProps()));
+            }
+        }
+
+        public void GrantRead(IIIdentityResource principal)
+        {
+            foreach (Bucket bucket in _buckets)
```

Comment:
var bucket

Reply:
I'm keeping `Bucket` in this instance, as the type of `_buckets` appears elsewhere in the file. See [CoreFX Coding Style](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md):

> 10. We only use `var` when it's obvious what the variable type is (e.g. `var stream = new FileStream(...)` not `var stream = OpenStandardInput()`).

Reply:
Still disagree, but I'll let it pass.